### PR TITLE
Fix info._repr_html_

### LINKED ISSUE #10471: mne.Info._repr_html_() does not show correct sensor locations?
I noticed that `mne.Info._repr_html_()` does not show the correct number of digitized sensor locations. Reproducible example (reprex):

```python
import mne

info = mne.create_info(["C3", "Cz", "C4"], 256, "eeg")
info.set_montage("biosemi64")

print(info)  # shows "dig: 6 items (3 Cardinal, 3 EEG)"
print(info._repr_html_())  # shows "0 points" for "Digitized points"
```

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #10471 (correctly report the number of digitized points available).

### CODE REVIEW

**Review by hoechenberger — APPROVED** [review]

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -172,6 +172,8 @@ Bugs
 
 - Add a video on how to operate the intracranial electrode contact location GUI in :ref:`tut-ieeg-localize` (:gh:`10800` by `Alex Rockhill`_)
 
+- Correctly report projections in :meth:`mne.io.meas_info.Info._repr_html_` (:gh:`10471` by `Clemens Brunner`_)
```
[the remark this answers is not in the document]
Comment (cbrnr) [reply doc/changes/latest.inc]: Better?

[the remark this answers is not in the document]
Comment (hoechenberger) [reply doc/changes/latest.inc]: Perfect!

**Review by hoechenberger — APPROVED** [review]

File: mne/io/meas_info.py [hunk 2]
```diff
@@ -1190,54 +1190,35 @@ def _repr_html_(self, caption=None):
...
-            ) + ' GMT'
-        else:
-            meas_date = None
+        meas_date = self.get('meas_date')
+        if meas_date is not None:
+            meas_date = meas_date.strftime("%B %d, %Y  %H:%M:%S") + ' GMT'
 
-        if 'projs' in self and self['projs']:
+        projs = self.get('projs')
+        if projs is not None and projs:
```
[the remark this answers is not in the document]
Comment (cbrnr) [reply mne/io/meas_info.py:1198]: OK, I usually explicitly test for `None`, but I guess in this case this is indeed better!

**Comment by cbrnr** [comment]
Checks are green except for the known macOS timeout, so this is ready!

**Comment by hoechenberger** [comment]
Thanks @cbrnr!

**Pull request merged**